packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -116,9 +129,28 @@ export const Webchat = forwardRef((props, ref) => {
     updateTheme({ ...webchatState.theme, ...props.theme })
   }, [props.theme])
 
+  const openMenu = () => {
+    if (menuIsOpened) {
+      setMenuIsOpened(false)
+    } else {
+      setMenuIsOpened(true)
+    }
+  }
+  const openEmoji = () => {
```

Comment:
if the method also closes the emoji , maybe we could call it toggleEmoji instead of openEmoji?

Reply:
sure, but i realize it doesn't have to be here